### PR TITLE
Added my part of story. Spanish

### PR DESCRIPTION
**Pull request by alexsimo**
This is part of the Udacity course exercise

### CODE REVIEW

**Comment by alexsimo** [comment]
Please review this, @casey-collab

File: espanol/pensamiento/guerra.md [hunk 1]
```diff
@@ -5,4 +5,5 @@ Ser
 
 Tras terminar con sus lamentaciones:
 
-[Una reflexi�n de gran importancia le asalt�](estoicismo/estoicismo.md)
\ No newline at end of file
+[Una reflexi�n de gran importancia le asalt�](estoicismo/estoicismo.md)
+[Se sentó en el suelo y se puso a meditar](meditacion/meditar.md)
```
Comment (casey-collab) [inline espanol/pensamiento/guerra.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by alexsimo** [comment]
Done! @casey-collab

**Comment by casey-collab** [comment]
Looks great! :checkered_flag:

**Pull request merged**